app/scripts/controllers/courses/CoursesShowCtrl.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+define(['paw'], function(paw) {
+  paw.controller('CoursesShowCtrl', ['$routeParams', function($routeParams) {
+    var _this = this;
+    var courseId = $routeParams.courseId; // For future Service calls
+
+    this.formatDate = function(finalExamDate) {
```

Comment:
This should not be here. Controllers don't have to know how data should be manipulated or styled. They should called [Angular's filters](https://docs.angularjs.org/guide/filter) to perform these kind of tasks.
Move this logic to a filter.